### PR TITLE
Added AWRSUtr

### PR DESCRIPTION
**Pull request by nikhilbharti**

### CODE REVIEW

File: src/main/scala/uk/gov/hmrc/domain/AwrsUtr.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.domain
+
+import play.api.libs.json.{Reads, Writes}
+
+case class AwrsUtr(utr: String) extends TaxIdentifier with SimpleName {
+  override lazy val toString = utr
```
Comment (andy-gray) [inline src/main/scala/uk/gov/hmrc/domain/AwrsUtr.scala]: We don't need another field here, make this a method

**Comment by applethepeanut** [comment]
Please do not merge yet. This is under discussion.

File: src/main/scala/uk/gov/hmrc/domain/AwrsUtr.scala (hunk 1 above)
Comment (nikhilbharti) [inline src/main/scala/uk/gov/hmrc/domain/AwrsUtr.scala]: changes are inline with how others have done. for example check Atedutr, CtUtr etc

Comment (andy-gray) [inline src/main/scala/uk/gov/hmrc/domain/AwrsUtr.scala]: Not any more...

**Comment by applethepeanut** [comment]
this is being tracked using https://jira.tools.tax.service.gov.uk/browse/GG-174

File: src/main/scala/uk/gov/hmrc/domain/AwrsUtr.scala (hunk 1 above)
Comment (nikhilbharti) [inline src/main/scala/uk/gov/hmrc/domain/AwrsUtr.scala]: Changed to def

**Pull request merged**